rosdep/base.yaml

```diff
@@ -3463,6 +3463,10 @@ pocketsphinx:
     xenial: [pocketsphinx]
     yakkety: [pocketsphinx]
     zesty: [pocketsphinx]
+polyclipping:
+  ubuntu:
+    trusty: [libpolyclipping-dev]
+    xenial: [libpolyclipping-dev]
```

Comment:
This package is available on all alive ubuntu distribution, please majke this entry distribution agnostic `ubuntu: [libpolyclipping-dev]`.
Also please add the corresponding debian entry